Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Models/GeneratedMethodModelBuilder.cs

```diff
@@ -35,7 +35,9 @@ public static class GeneratedMethodModelBuilder
             {
                 "System",
                 "System.Linq",
-                "System.Collections.Generic"
+                "System.Collections.Generic",
+                "System.Text",
+                "System.Reflection",
```

Comment:
We should not be needing reflection. That is the point of using source generators.

Reply:
Fixed